gapic-generator-java/src/main/java/com/google/api/generator/gapic/model/Method.java

```diff
@@ -99,6 +99,14 @@ public boolean isOperationPollingMethod() {
     return operationPollingMethod();
   }
 
+  public boolean isSupportedByTransport(Transport transport) {
```

Comment:
I know this may already be tested implicitly in some golden tests, since we refactored it to `Method` class, can we add a couple of true unit tests to `MethodTest`?

Reply:
Added some tests for this in the units.